woocommerce/woocommerce-ios
# Reviews tab: move view controllers away from Storyboard

Move Reviews VC from Storyboard, a prep step for #1838 

## Changes
- Moved 2 view controllers away from `Notifications.storyboard`:
  - The top-level view controller is just a navigation controller, an instance of `WooNavigationController`
  - The navigation controller's root view controller is `ReviewsViewController` that displays a list of reviews. its xib just contains a table view
- In `MainTabBarController`, added the Reviews tab manually similar to the Orders tab

## Testing

- Launch the app logged in
- Go to the Reviews tab --> it should display a list of reviews on the current store
- Go to the "My store" tab
- Go to settings, tap "Switch Store", and select a different store
- Go to the Reviews tab --> it should display a list of reviews on the new store
- Go to the "My store" tab
- Go to settings, tap "Log Out"
- Log in again to any store
- Go to the Reviews tab --> it should display a list of reviews on the current store

Update release notes:
